pilot/pkg/model/authorization.go

```diff
@@ -28,45 +28,56 @@ var (
 	rbacLog = istiolog.RegisterScope("rbac", "rbac debugging", 0)
 )
 
+type ServiceRoleConfig struct {
+	Name        string                 `json:"name"`
+	ServiceRole *rbacproto.ServiceRole `json:"service_role"`
+}
+
+type AuthorizationPolicyConfig struct {
+	Name                string                      `json:"name"`
+	Namespace           string                      `json:"namespace"`
+	AuthorizationPolicy *authpb.AuthorizationPolicy `json:"authorization_policy"`
+}
+
 // RolesAndBindings stores the the ServiceRole and ServiceRoleBinding in the same namespace.
 type RolesAndBindings struct {
 	// ServiceRoles in the same namespace.
-	Roles []Config
+	Roles []ServiceRoleConfig `json:"roles"`
 
 	// ServiceRoleBindings indexed by its associated ServiceRole's name.
-	Bindings map[string][]*rbacproto.ServiceRoleBinding
+	Bindings map[string][]*rbacproto.ServiceRoleBinding `json:"bindings"`
 }
 
 // AuthorizationPolicies organizes authorization policies by namespace.
 type AuthorizationPolicies struct {
```

Comment:
nit: Can we change this to something else? I think it's very easy to get confused with the AuthorizationPolicy CRD. Maybe rename it to AuthzPolicies?

Reply:
Yeah, maybe `processedAuthzPolicies` to align with the `processedAuthnPolicies`, should do in separate PR.